.travis.yml

```
@@ -46,7 +46,7 @@ install:
 
   # Summarise the environment
   # -------------------------
-  - conda list
+  - conda list --explicit
```

@corinnebosley I think you need to explicitly list the environment **before** `iris-grib` is added to it. This way you can reconstruct the environment _into which_ `iris-grib` may be installed. If you list it after `iris-grib` has been (pip-)installed, you might end up listing an environment that can't be automatically reconstructed. 

Perhaps try adding `--explicit` back to [L20](https://github.com/SciTools/iris-grib/blob/f954435fbaec39a1ed1ed493adde6611f217a4d1/.travis.yml#L20) and reverting the change here:

```suggestion
  - conda list
```